internal/api/apiservice/rebuild.go

```diff
@@ -158,38 +149,41 @@ func RebuildPackage(ctx context.Context, req schema.RebuildPackageRequest, deps
 			return nil, api.AsStatus(codes.Internal, errors.Wrap(err, "creating build definition repo reader"))
 		}
 		pth, _ := defs.Path(ctx, t)
-		buildDefLoc = rebuild.Location{
+		strat.buildDefLoc = rebuild.Location{
 			Repo: deps.BuildDefRepo.Repo,
 			Ref:  defs.Ref().String(),
 			Dir:  pth,
 		}
-		manualStrategy, err := defs.Get(ctx, t)
+		strat.ManualStrategy, err = defs.Get(ctx, t)
```

Comment:
oh shoot i just realized this was a poor name choice in my original code since it collides with the actual "rebuild.ManualStrategy" type. maybe "RepoStrategy"?